### PR TITLE
Route path for reset password included

### PR DESCRIPTION
**Pull request by Ade0211**
1) I  included the reset password route  
  2) I installed materialize-css in order to implement my M.toast
3) i changed the homepage path from (/front) to (/)

### CODE REVIEW

**Review by ISTANBULBEKLE — APPROVED** [review]
Great job guys

File: src/components/ResetPassword/resetpassword.jsx [hunk 1]
```diff
@@ -12,8 +12,8 @@ export default function ResetPassword() {
     setEmail(e.target.value);
   };
   const [email, setEmail] = useState("");
+  // eslint-disable-next-line
   const re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9] {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
   const PostData = () => {
     console.log("hi");
     if (re) {
```
Comment (Wolfjawan) [inline src/components/ResetPassword/resetpassword.jsx]: How this if statement checks the email?

File: src/components/ResetPassword/resetpassword.jsx [hunk 2]
```diff
@@ -12,8 +12,8 @@ export default function ResetPassword() {
     setEmail(e.target.value);
   };
   const [email, setEmail] = useState("");
+  // eslint-disable-next-line
   const re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9] {1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
   const PostData = () => {
     console.log("hi");
```
Comment (Wolfjawan) [inline src/components/ResetPassword/resetpassword.jsx]: console log should be removed

File: src/components/ResetPassword/resetpassword.jsx (hunk 1 above)
Comment (Ade0211) [reply src/components/ResetPassword/resetpassword.jsx]: I have changed the pattern in which the email received will be validated. Thanks for bringing my notice to this

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍🏼

**Review by Denis-Andrei — APPROVED** [review]
Looks good

**Pull request merged**